examples/pthreads/CIVL-0.17_2046/27_Boop_simple_vf_false.c

```diff
@@ -0,0 +1,83 @@
+#include <pthread.h>
```

Comment:
So in general we want to use the examples folder for nice illustrative examples. For instance, it would be good to place in the examples folder that standard bank account example. So please do that. I think Dekker would also be nice to keep in the examples folder maybe. I think that everything else should be moved to our regressions folder (test). So please restructure your files in such a way. Maybe in your ``test/pthread`` folder it would be good to prefix CIVL examples with ``civl-`` and svcomp example with `svcomp-`` or something like that.